plugins/network/whois

```diff
@@ -78,43 +80,48 @@ EOF
 # Args: domain name
 # Separated from graph_config so we can easily extend the plugin to support multiple domains.
 config() {
-	local NAME=${1}
+	local NAME
 	local FIELDNAME
+	for NAME in $DOMAINS
+	do
+		FIELDNAME="$(clean_fieldname "${NAME}")"
 
-	FIELDNAME="$(clean_fieldname "${NAME}")"
-
-	cat << EOF
-${FIELDNAME}.label expiry
+		cat << EOF
+${FIELDNAME}.label ${NAME}
 ${FIELDNAME}.cdef ${FIELDNAME},86400,/
 ${FIELDNAME}.warning ${WARNING}:
 ${FIELDNAME}.critical ${CRITICAL}:
 EOF
+	done
 }
 
 # Args: domain name
 fetch() {
-	local NAME=${1}
+	local NAME
 	local FIELDNAME
 	local CACHEFILE
 
-	FIELDNAME="$(clean_fieldname "${NAME}")"
+	for NAME in $DOMAINS
+	do
+		FIELDNAME="$(clean_fieldname "${NAME}")"
 
-	CACHEFILE="${MUNIN_PLUGSTATE}/$(basename "${0}").${FIELDNAME}.cache"
+		CACHEFILE="${MUNIN_PLUGSTATE}/$(basename "${0}").${FIELDNAME}.cache"
```

Comment:
I did not look at the new file, but the github visualization indicates, that you are mixing tabs and spaces here (and in some more lines below)?

Reply:
Nope, only tabs are used. I included existing code in a `for` loop and GitHub highlights tabs added for indentation.